tool/gravity/cli/clusterconfig.go

```diff
@@ -218,6 +221,42 @@ type configInitializer struct {
 	config   libclusterconfig.Interface
 }
 
+func validateUpdateConfig(localEnv *localenv.LocalEnvironment, config libclusterconfig.Interface) error {
```

Comment:
Nit: I would call this function `validateCloudConfig` or something to avoid confusion with regular update operation.